src/app/js/fields/FieldCompletes.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
the naming is weird... I'd prefer `FieldComplement` or `FieldAnnotation`. If it's too cumbersome, keep it like that, but try to avoid using verbs in active form in component names.